Index: homework9_symbo1.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\"\"\"homework9\"\"\"\n\n# Напишите программу, которая бы работала следующим образом -\n# находила символ \"#\" и если этот символ найден - удаляла предыдущий символ из строки.\n# Ваша задача обработать строки с \"#\" символом.\n#\n# Примеры:\n# \"a#bc#d\" ==>  \"bd\"\n# \"abc#d##c\"      ==>  \"ac\"\n# \"abc##d######\"  ==>  \"\"\n# \"#######\"       ==>  \"\"\n# \"\"              ==>  \"\"\n\n\ndef symbols(s):\n    i = 0\n    result = []\n    while i < len(s):\n        if s[i] == \"#\":\n            if result:\n                result.pop()\n        else:\n            result.append(s[i])\n        i += 1\n    return \"\".join(result)\n\n\n# Проверка\nprint(symbols(\"a#bc#d\"))\nprint(symbols(\"abc#d##c\"))\nprint(symbols(\"abc##d######\"))\nprint(symbols(\"#######\"))\nprint(symbols(\"\"))\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/homework9_symbo1.py b/homework9_symbo1.py
--- a/homework9_symbo1.py	
+++ b/homework9_symbo1.py	
@@ -1,7 +1,8 @@
 """homework9"""
 
 # Напишите программу, которая бы работала следующим образом -
-# находила символ "#" и если этот символ найден - удаляла предыдущий символ из строки.
+# находила символ "#" и если этот символ найден
+# - удаляла предыдущий символ из строки.
 # Ваша задача обработать строки с "#" символом.
 #
 # Примеры:
